docs/user-guide/helm.md

```diff
@@ -173,3 +173,20 @@ ENV HELM_PLUGINS="/home/argocd/.local/share/helm/plugins/"
 You have to remember about `HELM_PLUGINS` environment property - this is required to works plugins correctly.
 
 After that you have to use your custom image for ArgoCD installation.
+
+## Helm Version
```

Comment:
Do you think we should specify what the default version would be?

Reply:
It could make sense to mention that if `version` is omitted it will look at the `apiVersion` in Chart.yaml and decide which binary of Helm to use.

Reply:
Right!

Reply:
Ill fix it on both locations and push it again.

Reply:
Fixed in https://github.com/argoproj/argo-cd/commit/3011e10a69f5fa41c45cd329416d0500f1e19fe2